dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -1116,3 +1115,41 @@ def autogenerate(
     cmd = f"/opt/airflow/scripts/in_container/run_generate_migration.sh '{message}'"
     execute_command_in_shell(shell_params, project_name="db", command=cmd)
     fix_ownership_using_docker()
+
+
+@main.command(name="doctor", help="Troubleshoot breeze")
+@option_answer
+@click.pass_context
+def doctor(ctx):
```

Comment:
One more thing - addding `@dry_run` and `@verbose` will allow to print commands while executing or just showig what would have happened. This should work out-of-the-box with run_command and friends, but some care should be taken and `ifs` added to `rmdir` and similar.

Reply:
@potiuk Since most of the function calls work out of the box with the `@option_verbose` and `@option_dry_run` decorators, I have added these now to breeze doctor and seems to work as expected. Please review the changes and let me know your thoughts!